modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DERichTextField.testcase

```diff
@@ -135,7 +135,7 @@ definition {
 			fieldName = "Rich Text");
 	}
 
-	@description = "This is a stubbed description"
+	@description = "This is a test for LRQA-69427. This test verifies that data inputted on the Rich Text field persists."
```

Comment:
@guivilarim, will you use the same LRQA for all new Poshi tests? Is it how you guys in original Data Engine Team were proceeding?

Can we talk more about this procedure after our daily today?